lib/services/android-debug-service.ts

```diff
@@ -88,15 +69,7 @@ class AndroidDebugService implements IDebugService {
 		if (match) {
 			port = parseInt(match[1]);
 		} else {
-			let candidatePort = 40000;
-
-			for (; ! await this.isPortAvailable(candidatePort); ++candidatePort) {
-				if (candidatePort > 65534) {
-					this.$errors.failWithoutHelp("Unable to find free local port.");
-				}
-			}
-
-			port = candidatePort;
+			port = await getAvailablePort(40000);
```

Comment:
Because we want to start chrome devtools on port 40000 each time we start debugging. That's why we don't get random free port. If we get random free port, the users should copy the url each time `tns debug` is executed.